app/Console/Commands/MigrateSettingsData.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Organization;
+use App\Models\User;
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\Isolatable;
+
+use function Termwind\render;
+
+class MigrateSettingsData extends Command implements Isolatable
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:migrate-settings-data
+                            {--list : lists out available migrations}
+                            {--from=1.6.0 : when running all migrations, indicate which version the application is being migrated from. Previous migrations will be skipped.}
+                            {--migration= : a specific migration to run}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Migrates settings data that is not performed by database migrations; such as modifying the contents of database fields.';
+
+    protected $migrations = [
+        'EnableEngagementNotifications' => [
+            'version' => '1.7.0',
+            'handler' => 'enableEngagementNotificationsMigration',
+            'description' => 'Replaces older format of notifications_settings with only ["engagements" => "1"]. Setting the engagement notifications on be default. If the notifications_settings contains a valid engagements setting, then no changes are made.',
+        ],
+    ];
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $verbose = $this->options()['verbose'];
```

Comment:
```suggestion
       $verbose = $this->output->isVerbose();
```

See: https://github.com/laravel/framework/blob/b9342ff755dfea947e9d22687d0a0169d568b09e/src/Illuminate/Console/OutputStyle.php#L159-L167